cs3api4lab/tests/test_cs3_file_api.py

```
@@ -39,194 +37,134 @@ def test_read_file(self):
+                self.storage.stat(file_id, self.endpoint)
+        except:
+            self.storage.remove(file_id, self.endpoint)
 
     def test_read_directory(self):
-
         file_id = "/"
         read_directory = self.storage.read_directory(file_id, self.endpoint)
         self.assertIsNotNone(read_directory[0])
         self.assertIsNotNone(read_directory[0].path)
 
     def test_move_file(self):
-
         src_id = "/file_to_rename.txt"
         buffer = b"ebe5tresbsrdthbrdhvdtr"
-
         dest_id = "/file_after_rename.txt"
-
         try:
             self.storage.remove(dest_id)
-        except: pass
-
-        self.storage.write_file(src_id, buffer, self.endpoint)
-        self.storage.move(src_id, dest_id, self.endpoint)
-
-        self.storage.remove(dest_id, self.endpoint)
-        with self.assertRaises(IOError):
-            self.storage.stat(dest_id, self.endpoint)
+        except:
+            pass
+        try:
+            self.storage.write_file(src_id, buffer, self.endpoint)
+            self.storage.move(src_id, dest_id, self.endpoint)
+            self.storage.remove(dest_id, self.endpoint)
+            with self.assertRaises(IOError):
+                self.storage.stat(dest_id, self.endpoint)
+        except:
+            self.storage.remove(dest_id, self.endpoint)
 
     def test_move_file_already_exists(self):
         try:
             source_path = "/file_to_rename.txt"
             buffer = b"ebe5tresbsrdthbrdhvdtr"
-
             destination_path = "/file_after_rename.txt"
-
             self.storage.write_file(source_path, buffer, self.endpoint)
             self.storage.write_file(destination_path, buffer, self.endpoint)
-            
+
             with self.assertRaises(IOError) as context:
                 self.storage.move(source_path, destination_path, self.endpoint)
             self.assertEqual("file already exists", context.exception.args[0])
         finally:
+            self.storage.remove(source_path, self.endpoint)
```

should be try except:pass